README.md

```diff
@@ -104,7 +104,7 @@ cd mathmagicians-calculator
 npm i && npm start
 ```
 
-Then open [http://localhost:3000/](http://localhost:3000/) to see the app.
+Then open [http://localhost:3000/](https://math-magicians-calculator.herokuapp.com/) to see the app.
```

Comment:
Please try this one
```suggestion
Then open `http://localhost:3000/` to see the app.
```

Reply:
The task was when I will click on `http://localhost:3000/` this link it will open in a new tab and visit that link.
Is that correct?

Reply:
Please ask Ana Piris